events/templates/meetings/create_meeting.html

```diff
@@ -0,0 +1,110 @@
+    <link rel="stylesheet" type="text/css" href="{% static 'meetings/create_meeting.css' %}">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.0-2/css/all.min.css">
+
+    <title>Create Meeting </title>
+{% endblock %}
+
+<script>
+    total_meeting_dates_forms = "{{ total_meeting_forms }}"
+    total_participant_forms = "{{ total_particpant_forms }}"
+</script>
+
+{% block body-content %}
+<div class="container-shadow"></div>
+<div class="container">
+    <div class="wrap">
+        <div class="headings">
+            <a id="create-meeting" class="active"><span>{{ title }}:</span></a>
+        </div>
+        <form method="POST" >
+            {% csrf_token %}
+            {{create_event_form.as_p}}
+
+            <br/>
+            {% if optional_meetings_formset.non_form_errors %}
+                {{ optional_meetings_formset.non_form_errors }}
+            {% endif %}
+
+            {% if optional_meetings_formset.errors %}
+                {% for error in optional_meetings_formset.errors %}
+                    {{ error|escape }}
+                {% endfor %}
+            {% endif %}
+
+            {% if messages %}
+                <ul class="errorlist">
+                    {% for message in messages %}
+                        <li {% if message.tags %} class="{{ message.tags }}" {% endif %}>{{ message }}</li>
+                    {% endfor %}
+                </ul>
+            {% endif %}
+
+            <br/>
+            <h4>More optional meeting dates:</h4>
+            {{optional_meetings_formset.management_form}}
+            <div id="optional-meeting-dates-form-list">
+                {% if formset_meeting_data %}
+                    {% for form in formset_meeting_data %}
+                        <div id="form-{{form.id}}" class="optional-meeting-dates-form">
+                            <p>
+                                <label for="id_optional_meetings-{{form.id}}-date_time_start">Date time start:</label>
+                                <input type="datetime-local" value="{{form.date_time_start}}" name="optional_meetings-{{form.id}}-date_time_start" id="id_optional_meetings-{{form.id}}-date_time_start">
+                            </p>
+                            <p>
+                                <label for="id_optional_meetings-{{form.id}}-date_time_end">Date time end:</label>
+                                <input type="datetime-local" value="{{form.date_time_end}}" name="optional_meetings-{{form.id}}-date_time_end" id="id_optional_meetings-{{form.id}}-date_time_end">
+                            </p>
+                        </div>
+                    {% endfor %}
+                {% endif %}
+            </div>
+            
+            <div id="empty-optional-meeting-dates-form" class="hidden">{{ optional_meetings_formset.empty_form.as_p }}</div>
+            <button id="add-more-optional-meeting-dates" type="button" class="btn btn-dark">Add more dates</button>
```

Comment:
```suggestion
            <button id="add-more-optional-meeting-dates" type="button" class="btn btn-dark">
                <i class='far fa-calendar-plus' style='margin-right: 10px; font-size:17px; color: black;'></i>
                Add another date
            </button>```

Maybe you can add an icon that represents what the button does 😄

Reply:
Thanks, done :)